Features/Context/PlatformUI.php

```diff
@@ -78,6 +79,7 @@ protected function waitForLoadings()
         while ($page->find('css', $loadingSelector) != null) {
             usleep(100 * 1000); // 100ms
         }
+        sleep(1);
```

Comment:
I din't quite get why this is needed, it seem to already be matched above, and would another `usleep(100 * 1000)` be enough?